src/render/scene_optix.inl

```diff
@@ -686,12 +709,17 @@ Scene<Float, Spectrum>::ray_test_gpu(const Ray3f &ray, Mask active) const {
         const OptixConfig &config = optix_configs[s.config_index];
 
         UInt32 ray_mask(255),
-               ray_flags(OPTIX_RAY_FLAG_TERMINATE_ON_FIRST_HIT |
-                         OPTIX_RAY_FLAG_DISABLE_CLOSESTHIT),
+               ray_flags(OPTIX_RAY_FLAG_DISABLE_CLOSESTHIT),
```

Comment:
This loses the `OPTIX_RAY_FLAG_TERMINATE_ON_FIRST_HIT` optimization, which seems problematic.